ws_api/wealthsimple_api.py

```
@@ -339,15 +380,31 @@ def do_graphql_query(
             "x-ws-locale": "en-CA",
             "x-platform-os": "web",
         }
 
-        response_data = self.send_post(
-            url=self.GRAPHQL_URL, data=query, headers=headers
-        )
+        if self.session.session_id:
+            headers["x-ws-session-id"] = self.session.session_id
+
+        if self.session.access_token:
+            headers["Authorization"] = f"******"
```

Ruff (0.16.0)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)